### PR TITLE
adding Dawn keyboard support

### PR DESCRIPTION
**Pull request by dejaeghered**
## Description

Adding support for Dawn

## QMK Pull Request 

https://github.com/qmk/qmk_firmware/pull/11564

## Checklist

- [x] The VIA support for this keyboard is **MERGED** in QMK master already **(MANDATORY)**
- [x] The VIA definition follows the guide here: https://caniusevia.com/docs/layouts
- [x] I have a V3 JSON version for this keyboard definition.**(MANDATORY)**
- [x] I have tested this keyboard definition using VIA's "Design" tab.
- [x] I have tested this keyboard definition with firmware on a device.
- [x] I have assigned alpha keys and modifier keys with the correct colors.
- [x] The Vendor ID is not `0xFEED`

### CODE REVIEW

**Review by Xelus22 — CHANGES_REQUESTED** [review]
Please also create a V3 definition

**Comment by dejaeghered** [comment]
@Xelus22 added thanks!

File: v3/mechstudio/dawn.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+    "name": "Mech Studio Dawn",
+    "vendorId": "0x4d53",
+    "productId": "0x0004",
+    "matrix": { "rows": 6, "cols": 15 },
+    "keycodes": [],
+    "menus": [],
```
Comment (Xelus22) [inline v3/mechstudio/dawn.json:7]:
```suggestion
```

**Review by Xelus22 — CHANGES_REQUESTED** [review]

**Comment by dejaeghered** [comment]
confirmed

**Review by Xelus22 — APPROVED** [review]

**Pull request merged**